e2e/performance/app.test.js

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+'use strict';
+
+const env = require('../modules/environment');
+const {asyncSleep} = require('../modules/utils');
+
+describe('startup/app', function desc() {
+    this.timeout(30000);
+    this.slow(3000);
```

Comment:
@tboulis regarding the use of `.slow`, I see the same issues I've mentioned before (in using the e2e tests report):

1. The test/assertion relies on one run of the test, so it can be flaky
2. If the body of the test includes some setup steps (e.g. logging in etc), it will take those into account in its durations, too (this could be avoided if people writing performance tests are aware of this fact and include only the core of the test/the performance part in the test and everything else in before/after hooks, but it has a higher chance for errors perhaps).

Reply:
Nice :) If there's a way to run the same test multiple times and compare to their average, I'm 0/5 in terms of approaches. In that case I guess it won't be with slow though I guess? as it will need to get the outcome of the report, calculate averages or something and compare to something (so it needs to happen in the `checkReport` file that you have created?